pkg/controller/framework/controller_test.go

```diff
@@ -32,7 +32,12 @@ import (
 	"github.com/google/gofuzz"
 )
 
-func Example() {
```

Comment:
Please go read about example tests (https://blog.golang.org/examples), and then rethink the changes you're making with this PR.